kymatio/scattering1d/backend/numpy_backend.py

```diff
@@ -107,5 +107,19 @@ def transpose(cls, x):
         """Permute time and frequency dimension for time-frequency scattering"""
         return x.transpose(*list(range(x.ndim - 2)), -1, -2)
 
+    @classmethod
+    def conj_fr(cls, x, copy=True):
+        """Conjugate in frequency domain by swapping all bins (except dc);
+        assumes frequency along last axis.
+        """
+        if copy:
+            out = cls._np.zeros(x.shape, dtype=x.dtype)
+            out[..., 0] = x[..., 0]
+            out[..., 1:] = x[..., 1:][..., ::-1]
+            return out
+        else:
+            x[..., 1:] = x[..., 1:][..., ::-1]
```

Comment:
why do you need to implement this `copy=False` optional kwarg? unless there is need for both in JTFS, i'd be in favor of simplifying and leaving only the `copy=True` branch.

Reply:
More a future precaution as it's faster, but fair enough, removed